python/templates/Data.h.jinja2

```diff
@@ -19,8 +19,8 @@ public:
 {% endfor %}
 
 {% for member in VectorMembers + OneToManyRelations %}
-  unsigned int {{ member.name }}_begin{};
-  unsigned int {{ member.name }}_end{};
+  size_t {{ member.name }}_begin{};
```

Comment:
This would be a breaking change for generated datamodels. Is this strictly necessary?